### ISSUE TITLE
Conda Windows build issue

### ISSUE THREAD

**Issue by johnwason** [robotraconteur/robotraconteur#62]
@wolf @Tobias-Fischer

I am getting a conda build error on Windows that was not previously there:

```
gtest-md.lib(gtest-all.cc.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4

gtest-md.lib(gtest-all.cc.obj) : error LNK2001: unresolved external symbol __GSHandlerCheck_EH4

test\out\bin\robotraconteur_message_serialization_test.exe : fatal error LNK1120: 2 unresolved externals
```

Complete workflow:

https://github.com/robotraconteur/robotraconteur/actions/runs/4040771161/jobs/6948443654

Action file:

https://github.com/robotraconteur/robotraconteur/blob/01059ad2f3c47570cb728e191e017fb7cf7290f9/.github/workflows/main.yml#L528

**Comment by Tobias-Fischer** [comment]
My best bet is that you should use vs2019 instead of vs2017/2015 runtime that you currently install. The error is due to mismatching versions.

/cc @traversaro

**Comment by traversaro** [comment]
I think that while you are using VS2019, the toolset is being set to v141, i.e. (VS2017), see:
~~~
D:\a\robotraconteur\robotraconteur\build2>IF "1" == "1" (
IF "" == "" SET "CMAKE_GENERATOR_PLATFORM=x64"  
 IF "" == "" SET "CMAKE_GENERATOR_TOOLSET=v141" 
)
~~~

I suspect this is a bug of either https://github.com/conda-forge/vc-feedstock or the compilers metapackage, where installing `compilers` still targets VS2017's toolset, while the rest of conda-forge ecosystem targets vs2019, but I would need to double check.

**Comment by Tobias-Fischer** [comment]
This needs changing to vs2019-win64:
https://github.com/robotraconteur/robotraconteur/blob/01059ad2f3c47570cb728e191e017fb7cf7290f9/.github/workflows/main.yml#L542

**Comment by traversaro** [comment]
> This needs changing to vs2019_win-64:
> 
> https://github.com/robotraconteur/robotraconteur/blob/01059ad2f3c47570cb728e191e017fb7cf7290f9/.github/workflows/main.yml#L542

Good catch! Indeed I double checked and `compilers` installs the correct `vs2019_win-64` package.

**Comment by johnwason** [comment]
Thanks @traversaro, I also missed that!

**Comment by johnwason** [comment]
Fixed with #64 

The build was successful! Thanks guys!

But now llvm install script is failing on other tests 🙄

This package should be ready for conda-forge in a few weeks when I do the next release. Here is the feedstock: https://github.com/robotraconteur-packaging/robotraconteur-feedstock
